Assets/MeshLocalIdFix/MeshLocalIdFix.cs

```diff
@@ -0,0 +1,163 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+public static class MeshLocalIdFix
+{
+    // What is this ?
```

Comment:
// What is this ?
    // We changed the way LocalFileID are generated between Unity 2018.3 and 2019.1,
    // Unfortunately a bug was found later where Meshes with duplicated names in 2018.3
    // are imported with the new ID system in 2019.1 and thus all references to them are lost.
    // This was fixed in 2019.2.2f1 by ensuring that all duplicated Meshes created before 2019.1 keep their old ID.

    // The new emerging problem now is:
    // If you had a Mesh asset created in 2018.3 and migrated your project to 2019.1,
    // references to the imported Meshes end up using the new ID format for duplicated mesh names.
    // Upgrading to 2019.2.2f1 or later breaks all those references.
    // Unfortunately, there is no way for us to fix that internally without introducing breaking changes.

    // This script goes through all ModelImporters in the project and finds possible mismatching IDs.
    // Use it with care and make sure to save your project first because this is a destructive operation that cannot be reverted.
    // Also, it may take a lot of time to run
    // We'd recommend running it on a separate computer or overnight using the following command line:
    // unity.exe -quit -batchmode -executeMethod FixFBXFiles.DoFixReferences
    // see https://docs.unity3d.com/Manual/CommandLineArguments.html for more information.